SPLASH/src/game/states/LobbyHostState.cpp

```diff
@@ -17,9 +17,7 @@ LobbyHostState::~LobbyHostState() {
 }
 
 bool LobbyHostState::onMyTextInput(const ChatSent& event) {
-		Message temp{ NWrapperSingleton::getInstance().getMyPlayer().id, m_message };
-		addMessageToChat(temp);
-		m_network->sendChatMsg(m_message);
+
```

Comment:
This function is empty and isn't called from anywhere, remove it unless it's a work in progress